toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -2038,6 +2045,31 @@ func setGrubCfgSELinux(grubPath string, kernelCommandline configuration.KernelCo
 	return
 }
 
+func setGrubCfgFIPS(bootUUID, grubPath string, kernelCommandline configuration.KernelCommandLine) (err error) {
+	const (
+		enableFIPSPattern = "{{.FIPS}}"
+		enableFIPS        = "fips=1"
+		bootPrefix        = "boot="
+		uuidPrefix        = "UUID="
+	)
+
+	var fips string
+
+	// If EnableFIPS is set, add "fips=1 boot=UUID=<bootUUID value>" to the kernel cmdline in grub.cfg
+	if kernelCommandline.EnableFIPS {
+		fips = fmt.Sprintf("%s %s%s%s", enableFIPS, bootPrefix, uuidPrefix, bootUUID)
+	} else {
+		fips = ""
+	}
+
+	logger.Log.Debugf("Adding EnableFIPS('%s') to '%s'", fips, grubPath)
+	err = sed(enableFIPSPattern, fips, kernelCommandline.GetSedDelimeter(), grubPath)
+	if err != nil {
+		logger.Log.Warnf("Failed to set grub.cfg's EnableFIPS setting: %v", err)
+	}
+	return
```

Comment:
Small simplification and clarification of the variable's content:
```suggestion
	// If EnableFIPS is set, add "fips=1 boot=UUID=<bootUUID value>" to the kernel cmdline in grub.cfg
	fipsKernelArgument := ""
	if kernelCommandline.EnableFIPS {
		fipsKernelArgument = fmt.Sprintf("%s %s%s%s", enableFIPS, bootPrefix, uuidPrefix, bootUUID)
	}

	logger.Log.Debugf("Adding EnableFIPS('%s') to '%s'", fipsKernelArgument, grubPath)
	err = sed(enableFIPSPattern, fipsKernelArgument, kernelCommandline.GetSedDelimeter(), grubPath)
	if err != nil {
		logger.Log.Warnf("Failed to set grub.cfg's EnableFIPS setting: %v", err)
	}
	return
```